docker-compose.yml

```diff
@@ -4,37 +4,40 @@ services:
     ports:
       - 6333:6333
       - 6334:6334
-    expose:
-      - 6333
-      - 6334
-      - 6335
     networks:
       - ragapp-network
 
-  setup-qdrant:
-    image: curlimages/curl:latest
-    networks:
-      - ragapp-network
-    volumes:
-      - ./scripts/create_qdrant_collection.sh:/create_qdrant_collection.sh
-    command: /bin/sh -c "chmod +x /create_qdrant_collection.sh && /create_qdrant_collection.sh"
-    depends_on:
-      - qdrant
-
   ollama:
     image: ollama/ollama:latest
     ports:
       - 11434:11434 
     volumes:
       - ./ollama:/root/.ollama
-      - ./scripts/setup_ollama.sh:/setup_ollama.sh
-    entrypoint: /bin/bash
-    command: -c "chmod +x /setup_ollama.sh && /setup_ollama.sh"
     networks:
       - ragapp-network
+  
+  setup:
+    image: curlimages/curl:latest
+    networks:
+      - ragapp-network
+    volumes:
+      - ./scripts/create_qdrant_collection.sh:/create_qdrant_collection.sh
+      - ./scripts/setup_ollama.sh:/setup_ollama.sh
+    environment:
+      - MODEL=${MODEL:-phi3}
+      - OLLAMA_BASE_URL=http://ollama:11434
+      - QDRANT_URL=http://qdrant:6333
+      - COLLECTION_NAME=default
+      - DISTANCE_METRIC=Cosine
+      - VECTOR_SIZE=768
+    command: /bin/sh -c "chmod +x /create_qdrant_collection.sh /setup_ollama.sh && /create_qdrant_collection.sh && /setup_ollama.sh"
+    depends_on:
+      - qdrant
+      - ollama
 
   ragapp:
     build: .
+    image: ragapp/ragapp:latest
```

Comment:
isn't it either build or image?

Reply:
Yes, I added this so users can run the docker-compose with the YAML and scripts folder without cloning the repo to build the image.

Reply:
So, if we want to run with the current change, try: `docker-compose up --build` or with the latest released image, just run `docker-compose up`

Reply:
didn't know this works :D